### ISSUE TITLE
bug(example): README example not working

### ISSUE THREAD

**Issue by AmitMY** [tensorflow/model-card-toolkit#4]
## Expected Behavior
Example should work out of the box

## Actual Behavior
Error:
> AttributeError: module 'grpc.experimental.aio' has no attribute 'StreamUnaryCall'

## Steps to Reproduce the Problem

1. `pip install model-card-toolkit`
2. Try example:

```py
import model_card_toolkit

# Initialize the Model Card Toolkit with a path to store generate assets
model_card_output_path = "model_card"
mct = model_card_toolkit.ModelCardToolkit(model_card_output_path)

# Initialize the model_card_toolkit.ModelCard, which can be freely populated
model_card = mct.scaffold_assets()
model_card.model_details.name = 'My Model'

# Write the model card data to a JSON file
mct.update_model_card_json(model_card)

# Return the model card document as an HTML page
html = mct.export_format()
```

## Specifications

- Version: 0.1.0
- Platform: centos

##  Full error trace:
```
Traceback (most recent call last):
  File "/home/nlp/amit/PhD/SpeakerDetection/simple_tf/tools/model_card.py", line 1, in <module>
    import model_card_toolkit
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/model_card_toolkit/__init__.py", line 16, in <module>
    from model_card_toolkit.model_card_toolkit import ModelCardToolkit
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/model_card_toolkit/model_card_toolkit.py", line 32, in <module>
    from model_card_toolkit.utils import graphics
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/model_card_toolkit/utils/graphics.py", line 26, in <module>
    import tensorflow_model_analysis as tfma
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/tensorflow_model_analysis/__init__.py", line 23, in <module>
    from tensorflow_model_analysis.api import tfma_unit as test
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/tensorflow_model_analysis/api/tfma_unit.py", line 66, in <module>
    import apache_beam as beam
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/apache_beam/__init__.py", line 101, in <module>
    from apache_beam import io
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/apache_beam/io/__init__.py", line 39, in <module>
    from apache_beam.io.gcp.pubsub import *
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/apache_beam/io/gcp/pubsub.py", line 48, in <module>
    from google.cloud import pubsub
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/cloud/pubsub.py", line 20, in <module>
    from google.cloud.pubsub_v1 import PublisherClient
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/cloud/pubsub_v1/__init__.py", line 18, in <module>
    from google.cloud.pubsub_v1 import publisher
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/cloud/pubsub_v1/publisher/__init__.py", line 17, in <module>
    from google.cloud.pubsub_v1.publisher.client import Client
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/cloud/pubsub_v1/publisher/client.py", line 29, in <module>
    from google.cloud.pubsub_v1.gapic import publisher_client
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/cloud/pubsub_v1/gapic/publisher_client.py", line 28, in <module>
    import google.api_core.gapic_v1.client_info
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/api_core/gapic_v1/__init__.py", line 26, in <module>
    from google.api_core.gapic_v1 import method_async  # noqa: F401
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/api_core/gapic_v1/method_async.py", line 20, in <module>
    from google.api_core import general_helpers, grpc_helpers_async
  File "/home/nlp/amit/anaconda2/envs/meta-scholar/lib/python3.7/site-packages/google/api_core/grpc_helpers_async.py", line 145, in <module>
    class _WrappedStreamUnaryCall(_WrappedUnaryResponseMixin, _WrappedStreamRequestMixin, aio.StreamUnaryCall):
AttributeError: module 'grpc.experimental.aio' has no attribute 'StreamUnaryCall'
```

**Comment by AmitMY** [comment]
Workaround:
```
pip install google-api-core==1.16.0
```

**Comment by shuklak13** [comment]
Looks like this is the same issue as https://github.com/googleapis/python-api-core/issues/40, which was fixed with `pip install google-api-core==1.16.0`.
